examples/line.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import vedo
+
+vedo.settings.default_backend = "vtk"
+
+from brainrender import Scene
+from brainrender.actors import Points, Line
+
+# Display the Allen Brain mouse atlas at 25 micrometer resolution.
+scene = Scene(atlas_name="allen_mouse_25um")
+
+# Highlight the cerebral cortex; 688 is the corresponding Allen Brain Atlas ID.
+scene.add_brain_region(688, alpha=0.2, color="green")
```

Comment:
Just to match the other examples, and show a slightly more intuitive API, could you replace the ID with the abbreviation? I hope users don't have to remember IDs to use brainrender!

Reply:
What is the abbreviation? Allen SDK works with IDs and proper names, so those are the only ones I know.

Reply:
```suggestion
scene.add_brain_region("CTX", alpha=0.2, color="green")
```

Reply:
Addressed in 066ec5d.